src/10_functions.py

```diff
@@ -1,6 +1,16 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even(n):
+    if n % 2 == 0:
+        return True
+        pass
+    else:
+        return False
+        pass
```

Comment:
When a function run `return` nothing after the return statement will be run so you don't need `pass` here :)